wdl/ClusterBatch.wdl

```diff
@@ -282,6 +285,14 @@ workflow ClusterBatch {
     }
   }
 
+  call sv_counts.PlotSVCountsPerSample {
+    input:
+      prefix = batch,
+      vcfs = [ClusterDepth.clustered_vcf, ClusterPESR_manta.clustered_vcf, ClusterPESR_wham.clustered_vcf, ClusterPESR_melt.clustered_vcf, ClusterPESR_scramble.clustered_vcf],
+      N_IQR_cutoff = N_IQR_cutoff_plotting,
+      sv_pipeline_docker = sv_pipeline_docker
```

Comment:
Can you also add the runtime_attr override parameters for the tasks in PlotSvCountsPerSample, here and in FilterBatchSites? This will require you to make those workflow inputs as well as pass them when you call the subworkflow